### PR TITLE
Fix: Eval as boolean

### PR DESCRIPTION
**Pull request by XVincentX**
The current code will fail and fallback to default options values in case the `minItems` or `maxItems` is 0, which is not what we want. (0 evaluates as false)

I have tried to write a test for it but apparently the helpers I found are not good enough. I do not know how to assert that the length of a schema-generated array is not empty, nor a way to say "it has _at least_ X elements"

### CODE REVIEW

**Comment by XVincentX** [comment]
@peterwald

**Comment by pateketrueke** [comment]
> I have tried to write a test for it but apparently the helpers I found are not good enough. I do not know how to assert that the length of a schema-generated array is not empty, nor a way to say "it has at least X elements"

Yeah, I think we could add a `minLength` around: https://github.com/json-schema-faker/json-schema-faker/blob/master/tests/schema/helpers.js#L94

File: src/lib/types/array.js [hunk 1]
```diff
@@ -63,13 +63,13 @@ function arrayType(value, path, resolve, traverseCallback) {
 
   if (optionAPI('minItems')) {
     // fix boundaries
-    minItems = !maxItems
+    minItems = maxItems === undefined
```
Comment (pateketrueke) [inline src/lib/types/array.js:66]: I think could test like `typeof maxItems  ===  'undefined'` instead

**Review by pateketrueke — DISMISSED** [review]

File: src/lib/types/array.js (hunk 1 above)
Comment (XVincentX) [reply src/lib/types/array.js]: Done in 65c9366

**Comment by XVincentX** [comment]
@pateketrueke 

Tests added! If this is ready and could be releases, it'd be great 🙏

**Review by pateketrueke — APPROVED** [review]

**Comment by XVincentX** [comment]
@pateketrueke Is this going to be merged/released automatically? What's the procedure?

**Comment by pateketrueke** [comment]
I was waiting for TravisCI to complete, thank you!

**Comment by XVincentX** [comment]
Ay perfecto!

**Pull request merged**